src/main/gui/main-window.ts

```diff
@@ -379,6 +368,28 @@ export const createMainWindow = async (args: OpenArguments) => {
             return;
         }
 
+        if (isMac) {
+            if (process.env.STARTUPFILE) {
+                // Open file with chaiNNer on other platforms
+                const result = openSaveFile(process.env.STARTUPFILE);
+                ipcMain.handle('get-cli-open', () => result);
```

Comment:
Would it be possible to move the whole `if (isMac)` block back into `registerEventHandlerPreSetup`?

These ICP handlers must be registered for the frontend to be able to call them, and right now, it is not clear that `get-cli-open` is always handled.

Reply:
Done.